### ISSUE TITLE
New update: Hal Daumé III - new citations

### ISSUE THREAD

**Issue by postmillenial** [postmillenial/reading-list#1217]
From: scholaralerts-noreply@google.com<br>
Subject: Hal Daum&eacute; III - new citations<br>
First Attachment: http://ift.tt/19fNRe4<br>
Date: November 02, 2017 at 02:30AM<br><br><div style="font-family: arial,sans-serif; font-size: medium;">
<h3 style="font-weight: normal; margin: 0; font-size: medium;"><a href="http://ift.tt/2zZJXsx"><font size="3" color="#1A0DAB">A Sensor-Driven Hierarchical Method for Domain Adaptation in Classification of Remote Sensing Images</font></a></h3>
<div><font size="2" color="#006621">C Paris, L Bruzzone&nbsp;- IEEE Transactions on Geoscience and Remote&nbsp;&hellip;, 2017</font></div>
<div><font size="2" color="#222222">This paper presents a sensor-driven hierarchical domain adaptation method that aims at<br>
transferring the knowledge from a source domain (RS image where reference data are<br>
available) to a different but related target domain (RS image where no labeled reference<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2hwQVyr" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2zYgW0j" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2hwA3rz" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2zZRqaW"><font size="3" color="#1A0DAB">Attentional Action-Driven Deep Networks for Visual Object Tracking</font></a>
</h3>
<div><font size="2" color="#006621">&#50980;&#49345;&#46160; - 2017</font></div>
<div><font size="2" color="#222222">This dissertation proposes a novel visual tracking method which is controlled by sequential<br>
actions learned by deep reinforcement learning. In the recent trackers using deep networks,<br>
tracking-by-detection scheme is adopted to select the target position with the highest<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2hxD22N" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2zZxtkJ" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2hwL382" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><h3 style="font-weight: normal; margin: 0; font-size: medium;"><a href="http://ift.tt/2A02Lry"><font size="3" color="#1A0DAB">A Review on Quantification Learning</font></a></h3>
<div><font size="2" color="#006621">P Gonz&aacute;lez, A Casta&ntilde;o, NV Chawla, JJD Coz&nbsp;- ACM Computing Surveys (CSUR), 2017</font></div>
<div><font size="2" color="#222222">Abstract The task of quantification consists in providing an aggregate estimation (eg, the<br>
class distribution in a classification problem) for unseen test sets, applying a model that is<br>
trained using a training set with a different data distribution. Several real-world applications<br></font></div>
<table><tbody><tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2hwXRvh" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2zYy2v2" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2hwMLq8" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr></tbody></table>
<br><hr noshade size="1">
<font size="2" color="#222222">This Google Scholar Alert is brought to you by Google.<br><br><a href="http://ift.tt/2vUKjPi"><font color="#1A0DAB">Cancel alert</font></a><br><a href="http://ift.tt/2eWhNpX"><font color="#1A0DAB">List my alerts</font></a></font>
</div>
<br><br>
via Gmail
